internal/install/recipe_installer.go

```diff
@@ -156,7 +156,7 @@ func (i *RecipeInstaller) Install() error {
 		i.status.InstallCanceled()
 		return nil
 	case err = <-errChan:
-		if err == types.ErrInterrupt {
+		if errors.Is(err, types.ErrInterrupt) {
```

Comment:
`ooh_nice.gif`

Reply:
There's a bit we can do to make this generally better throughout the project.